numpy/lib/function_base.py

```diff
@@ -4812,26 +4812,39 @@ def trapz(y, x=None, dx=1.0, axis=-1):
 
     Examples
     --------
-    >>> np.trapz([1,2,3])
+    Use the trapezoidal rule on evenly spaced points with different syntax for
+    the point spacing:
+
+    >>> np.trapz([1, 2, 3])
     4.0
-    >>> np.trapz([1,2,3], x=[4,6,8])
+    >>> np.trapz([1, 2, 3], x=[4, 6, 8])
     8.0
-    >>> np.trapz([1,2,3], dx=2)
+    >>> np.trapz([1, 2, 3], dx=2)
     8.0
```

Comment:
Suggestion: since this is the first example, mentioning the point spacing before showing it is a bit confusing (took me a while to understand where the point spacing was before I noticed that's what the `x` in the second example was for). So this could read:

```
Use the trapezoidal rule on evenly spaced points:

>> np.trapz([1, 2, 3])
4.0

The spacing between sample points can be selected by either the ``x`` or ``dx`` arguments:

>> np.trapz([1, 2, 3], x=[4, 6, 8])
8.0
>> np.trapz([1, 2, 3], dx=2)
8.0
```

Reply:
Thanks, I like those suggestions.  If you are happy, will probably merge it all as is soon.